.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,28 @@
+## Steps to Reproduce
+
+1. 
+2. 
+3. 
+
+## Expected Behavior
+
+Please describe the behavior you are expecting
+
+## Current Behavior
+
+What is the current behavior?
+
+### On which platforms did you notice this
+
+- [ ] MacOS
+- [ ] Linux 
+- [ ] Windows
+
+**Version Used**:
```

Comment:
I'd rather instruct them to specifically run `mono --version` so we get full commit hash etc. which is included there.

Reply:
Speaking of, why don't we make `mono --version` output OS in addition to architecture? Then we wouldn't need the OS list above either.

Reply:
Made small tweak but `mono --version` is useful only for people running Mono directly.